### ISSUE TITLE
Get multiple arsenal tables on the same html document

### ISSUE THREAD

**Issue by jcutler79** [mayoverse/arsenal#307]
Hi,

I have been trying to make a single html document with many cross tables / contingency tables. Arsenal is the best package I have found for making these tables, and I am able to make a single pdf with all the tables I want. But I was wondering, is there a way to make a single html document with multiple tables in it? 

Using write2html, I can make an individual html file with a single table in it (no other tables, no text or other elements), but this isn't what I need. Is there a way to do this with, perhaps, xtable or something else? I've been looking at the write2 tutorial and trying to make it work, but am so far unsuccessful at doing anything other than creating individual html files for each table separately.

Thank you,
James Cutler

**Comment by eheinzen** [comment]
I hope you got this figured out?  There are two ways:

1. If you want to use `write2html`, you can put all your tables in a `list()`, and they'll get printed one-after-the-other.
2. Otherwise, you can always use R Markdown directly, by using a .Rmd document

**Comment by sinnweja** [comment]
I've been able to do something like this, but instead of the cat() using just text, you may be able to put html headers, or something else that separates the tables.
`
example(arsenal);

write2html(list(cat("Tab3\n"), summary(tab3, digits=3), cat("Tab.test\n"), summary(tab.test, digits=2)), file="twotables.html")
`

**Comment by jcutler79** [comment]
Yes, thank you! I did get it figured out. I realized I didn't even need to
ask. I got confused and thought arsenal was only compatible with making pdf
documents.

You wouldn't happen to know how to make the tables wider? Some of the cells
have the count on top of the parenthetical percentage, but I'm wondering if
I can widen the tables and get all of the cells to have counts and percents
side by side on the same line.

Thanks again,
James

**Comment by eheinzen** [comment]
I've always found CSS to be helpful with that.  Something like

```
library(arsenal)
data(mockstudy)

write2html(list(
  "<style>.main-container {max-width: 1600px; width: 1600px;}</style>",
  tableby(sex ~ age, data = mockstudy)
), file = "test.html")
```

**Comment by jcutler79** [comment]
That looks really promising. However, I tried this

.main-container{
  max-width: 1600px;
  width: 1600px;
}

in my css file, and it didn't work. I tried some other widely ranging
widths, and it doesn't change the table width at all. I guess I should
explain that I have realized what I wanted all along was to simply create a
regular old html document in rmarkdown, and everything is working totally
perfect with that and including arsenal tables in the document is working
great, so I just decided to go with that approach. The only thing now is
that the tables' width isn't changing.

Is the class selector going to be .main-container in this case, given that
I'm just putting my tables in code chunks in an rmarkdown html document? Or
would the selector be something else?

Thank you,
James
